### ISSUE TITLE
Creating the API for PostSpectator

### ISSUE THREAD

**Issue by ibrahimbayat** [bounswe/2021SpringGroup2#53]
Creating the API for PostSpectator

**Comment by ibrahimbayat** [comment]
I created the API function of spectator post with using an api which generates random person. It adds a spectator to an event. It will be changed according to our database system.

**Comment by ibrahimbayat** [comment]
Integrating the api to the database system was a very hard period for me. Because it is very hard for me downloading, making setup and using sqlalchemy and docker. I am using these tools for the first time. Firstly to setup and use database system has taken my two or three hours, later I learned that there were problems in our database system and database system has a lot of bugs and it was been updating by my friends. I had to  fix this myself in my computer ( sometimes even upper case, lower case letters mistakes). Then I watched git ps videos and docker videos to use these in my api's and to push changes, open branches. I watched  a lot of videos from the web also for git and docker. My code succesfully adding spectator to an event with taking a user from database. If database has no user with given id it creates a random user with a third party api (https://pipl.ir/v1/getPerson). I tested this api with adding users and events to the our database and I sended post signal with using Postman. Then I opened a branch named postspectator. I committed and pushed mychanges to this branch. Actually just adding tuples to databases take many hours because I am using these tools for the first time. I got very much help  from my friends Berkay and Ege while writing my API's.

**Comment by ibrahimbayat** [comment]
After the review of my friends, I  made changes for Blueprint objects. While pushing this function to the "postspectator" branch I had pushed the .pyc files by mistake. While making changes for the .pyc files I deleted  all files with the .pyc files by mistake.  With the help of my friend, I could restore the files and pushed it to the "postspectator" branch. Then we merged "postspectator" branch with the "development" branch.
